.drone.yml

```diff
@@ -8,8 +8,28 @@ trigger:
     include:
       - 'refs/tags/v*'
       - 'refs/heads/main'
+      - 'refs/heads/DOP-*'
 
 steps:
+  - name: publish-dev
+    image: plugins/ecr
+    settings:
+      create_repository: true
+      registry: 795250896452.dkr.ecr.us-east-1.amazonaws.com
+      repo: 795250896452.dkr.ecr.us-east-1.amazonaws.com/docs/${DRONE_REPO_NAME}
+      tags:
+        - git-${DRONE_COMMIT_SHA:0:7}
+        - latest
```

Comment:
(Non-blocking recommendation) I don't think we currently use the `latest` image tag for anything. It might be good to avoid tagging dev images with `latest` in case it causes any unwanted conflicts in the future. This overlap in image tag names was actually an issue on Hapley once upon a time ([PR](https://github.com/mongodb/docs-hapley/pull/97)).

Reply:
good catch! left the `latest` image tag strictly to prod and updated the tags for dev/stage